src/wp-admin/css/about.css

```diff
@@ -611,25 +580,47 @@
 /* 1.3 - Header */
 
 .about__header {
+	position: relative;
 	margin-bottom: var(--gap);
 	padding-top: 0;
-	background-position: center;
-	background-repeat: no-repeat;
-	background-size: cover;
-	background-image: url('../images/about-header-about.svg');
+	background-image: url('../images/about-texture.png');
 	background-color: var(--accent-2);
+	background-size: 500px 500px;
+	background-blend-mode: overlay;
 	color: var(--text-light);
 }
 
-.credits-php .about__header {
+.about__header:before {
```

Comment:
While `:` and `::` are interchangeable for compatibility reasons, `:` is legacy and `::` was introduced in CSS3. The PR changes an already existing instance of `::` to `:` and continues to use `:` throughout. Is there a reason why we are changing to `:`?

Reply:
Bad habit - thanks for catching :)

Reply:
Not an issue per se, I just saw the change and was curious. It's by no means the only use of `:` in Core!